### ISSUE TITLE
Creating an Alert-driven SLO fails

### ISSUE THREAD

**Issue by danifitz** [newrelic/nr1-slo-r#63]
## Description

Trying to create an alert-driven SLO based upon Capacity. [See screenshot](https://imgur.com/a/hikl79x)

## Steps to Reproduce

Create a new SLO based upon Capacity, fill in the fields, add the name of an alert policy found in SLOR_ALERTS table in Insights

## Expected Behaviour

An SLO is created

## Relevant Logs / Console output

Output from JS Console:
```
6single-document.js:25 Uncaught (in promise) TypeError: _alerts.map is not a function
    at _getAlertsWhereClause (single-document.js:25)
    at _getOpenedAlertNRQL (single-document.js:33)
    at _getAlertDrivenSLOData$ (single-document.js:65)
    at tryCatch (runtime.js:45)
    at Generator.invoke [as _invoke] (runtime.js:271)
    at Generator.prototype.<computed> [as next] (runtime.js:97)
    at tryCatch (runtime.js:45)
    at invoke (runtime.js:135)
    at runtime.js:170
    at new Promise (<anonymous>)
_getAlertsWhereClause @ single-document.js:25
_getOpenedAlertNRQL @ single-document.js:33
_getAlertDrivenSLOData$ @ single-document.js:65
tryCatch @ runtime.js:45
invoke @ runtime.js:271
prototype.<computed> @ runtime.js:97
tryCatch @ runtime.js:45
invoke @ runtime.js:135
(anonymous) @ runtime.js:170
callInvokeWithMethodAndArg @ runtime.js:169
enqueue @ runtime.js:192
prototype.<computed> @ runtime.js:97
push.../../../../.local/share/newrelic-cli/client/1.10.10/node_modules/regenerator-runtime/runtime.js.exports.async @ runtime.js:216
_getAlertDrivenSLOData @ single-document.js:52
query$ @ single-document.js:320
tryCatch @ runtime.js:45
invoke @ runtime.js:271
prototype.<computed> @ runtime.js:97
tryCatch @ runtime.js:45
invoke @ runtime.js:135
(anonymous) @ runtime.js:170
callInvokeWithMethodAndArg @ runtime.js:169
enqueue @ runtime.js:192
prototype.<computed> @ runtime.js:97
push.../../../../.local/share/newrelic-cli/client/1.10.10/node_modules/regenerator-runtime/runtime.js.exports.async @ runtime.js:216
query @ single-document.js:311
(anonymous) @ slo-list.js:100
(anonymous) @ slo-list.js:92
loadData$ @ slo-list.js:89
tryCatch @ runtime.js:45
invoke @ runtime.js:271
prototype.<computed> @ runtime.js:97
tryCatch @ runtime.js:45
invoke @ runtime.js:135
(anonymous) @ runtime.js:170
callInvokeWithMethodAndArg @ runtime.js:169
enqueue @ runtime.js:192
prototype.<computed> @ runtime.js:97
push.../../../../.local/share/newrelic-cli/client/1.10.10/node_modules/regenerator-runtime/runtime.js.exports.async @ runtime.js:216
loadData @ shared~dbd1e6a7-f021-472a-9575-b554a6908aa7--nr1-slo-r-nerdlet~dbd1e6a7-f021-472a-9575-b554a6908aa7-~42f9f94e.js:1768
(anonymous) @ slo-list.js:82
callCallback @ react-dom.development.js:11288
commitUpdateEffects @ react-dom.development.js:11327
commitUpdateQueue @ react-dom.development.js:11315
commitLifeCycles @ react-dom.development.js:16115
commitAllLifeCycles @ react-dom.development.js:17386
callCallback @ react-dom.development.js:143
invokeGuardedCallbackDev @ react-dom.development.js:193
invokeGuardedCallback @ react-dom.development.js:250
commitRoot @ react-dom.development.js:17582
completeRoot @ react-dom.development.js:19036
performWorkOnRoot @ react-dom.development.js:18965
performWork @ react-dom.development.js:18873
performSyncWork @ react-dom.development.js:18847
requestWork @ react-dom.development.js:18716
scheduleWork @ react-dom.development.js:18525
enqueueSetState @ react-dom.development.js:12437
Component.setState @ react.development.js:460
upsertDocumentInList @ index.js:234
upsertDocumentCallback$ @ index.js:188
tryCatch @ runtime.js:45
invoke @ runtime.js:271
prototype.<computed> @ runtime.js:97
tryCatch @ runtime.js:45
invoke @ runtime.js:135
(anonymous) @ runtime.js:170
callInvokeWithMethodAndArg @ runtime.js:169
enqueue @ runtime.js:192
prototype.<computed> @ runtime.js:97
push.../../../../.local/share/newrelic-cli/client/1.10.10/node_modules/regenerator-runtime/runtime.js.exports.async @ runtime.js:216
upsertDocumentCallback @ index.js:182
writeNewSloDocument$ @ slo-form.js:218
tryCatch @ runtime.js:45
invoke @ runtime.js:271
prototype.<computed> @ runtime.js:97
tryCatch @ runtime.js:45
invoke @ runtime.js:135
(anonymous) @ runtime.js:145
Promise.then (async)
(anonymous) @ slo-list.js:104
(anonymous) @ slo-list.js:92
loadData$ @ slo-list.js:89
tryCatch @ runtime.js:45
invoke @ runtime.js:271
prototype.<computed> @ runtime.js:97
tryCatch @ runtime.js:45
invoke @ runtime.js:135
(anonymous) @ runtime.js:170
callInvokeWithMethodAndArg @ runtime.js:169
enqueue @ runtime.js:192
prototype.<computed> @ runtime.js:97
push.../../../../.local/share/newrelic-cli/client/1.10.10/node_modules/regenerator-runtime/runtime.js.exports.async @ runtime.js:216
loadData @ shared~dbd1e6a7-f021-472a-9575-b554a6908aa7--nr1-slo-r-nerdlet~dbd1e6a7-f021-472a-9575-b554a6908aa7-~42f9f94e.js:1768
(anonymous) @ slo-list.js:82
callCallback @ react-dom.development.js:11288
commitUpdateEffects @ react-dom.development.js:11327
commitUpdateQueue @ react-dom.development.js:11315
commitLifeCycles @ react-dom.development.js:16115
commitAllLifeCycles @ react-dom.development.js:17386
callCallback @ react-dom.development.js:143
invokeGuardedCallbackDev @ react-dom.development.js:193
invokeGuardedCallback @ react-dom.development.js:250
commitRoot @ react-dom.development.js:17582
completeRoot @ react-dom.development.js:19036
performWorkOnRoot @ react-dom.development.js:18965
performWork @ react-dom.development.js:18873
performSyncWork @ react-dom.development.js:18847
requestWork @ react-dom.development.js:18716
scheduleWork @ react-dom.development.js:18525
enqueueSetState @ react-dom.development.js:12437
Component.setState @ react.development.js:460
upsertDocumentInList @ index.js:234
upsertDocumentCallback$ @ index.js:188
tryCatch @ runtime.js:45
invoke @ runtime.js:271
prototype.<computed> @ runtime.js:97
tryCatch @ runtime.js:45
invoke @ runtime.js:135
(anonymous) @ runtime.js:170
callInvokeWithMethodAndArg @ runtime.js:169
enqueue @ runtime.js:192
prototype.<computed> @ runtime.js:97
push.../../../../.local/share/newrelic-cli/client/1.10.10/node_modules/regenerator-runtime/runtime.js.exports.async @ runtime.js:216
upsertDocumentCallback @ index.js:182
writeNewSloDocument$ @ slo-form.js:218
tryCatch @ runtime.js:45
invoke @ runtime.js:271
prototype.<computed> @ runtime.js:97
tryCatch @ runtime.js:45
invoke @ runtime.js:135
(anonymous) @ runtime.js:145
Promise.then (async)
invoke @ runtime.js:144
(anonymous) @ runtime.js:170
callInvokeWithMethodAndArg @ runtime.js:169
enqueue @ runtime.js:192
prototype.<computed> @ runtime.js:97
push.../../../../.local/share/newrelic-cli/client/1.10.10/node_modules/regenerator-runtime/runtime.js.exports.async @ runtime.js:216
writeNewSloDocument @ shared~dbd1e6a7-f021-472a-9575-b554a6908aa7--nr1-slo-r-nerdlet~dbd1e6a7-f021-472a-9575-b554a6908aa7-~42f9f94e.js:816
upsertHandler @ slo-form.js:204
value @ wanda--wanda-ec-ui--third-party-sdk-release-844.js:185
callCallback @ react-dom.development.js:143
invokeGuardedCallbackDev @ react-dom.development.js:193
invokeGuardedCallback @ react-dom.development.js:250
invokeGuardedCallbackAndCatchFirstError @ react-dom.development.js:264
executeDispatch @ react-dom.development.js:555
executeDispatchesInOrder @ react-dom.development.js:577
executeDispatchesAndRelease @ react-dom.development.js:674
executeDispatchesAndReleaseTopLevel @ react-dom.development.js:682
forEachAccumulated @ react-dom.development.js:656
runEventsInBatch @ react-dom.development.js:810
runExtractedEventsInBatch @ react-dom.development.js:818
handleTopLevel @ react-dom.development.js:4923
batchedUpdates$1 @ react-dom.development.js:19056
batchedUpdates @ react-dom.development.js:2148
dispatchEvent @ react-dom.development.js:5002
interactiveUpdates$1 @ react-dom.development.js:19111
interactiveUpdates @ react-dom.development.js:2167
dispatchInteractiveEvent @ react-dom.development.js:4979
Show 100 more frames
6single-document.js:105 Uncaught (in promise) TypeError: Cannot read property 'actor' of null
    at _getAlertDrivenSLOData$ (single-document.js:105)
    at tryCatch (runtime.js:45)
    at Generator.invoke [as _invoke] (runtime.js:271)
    at Generator.prototype.<computed> [as next] (runtime.js:97)
    at tryCatch (runtime.js:45)
    at invoke (runtime.js:135)
    at runtime.js:145
```

## Your Environment

[TIP]:  # ( Include as many relevant details about your environment as possible. )

* NR1 CLI version used:
@datanerd/nr1/1.10.10 darwin-x64 node-v10.16.3
* Browser name and version:
Chrome 79
* Operating System and version:
Mac OS 10.14.3

**Comment by danifitz** [comment]
I believe this is caused due to an error earlier in slo-form.js component where the Multiselect component complains it's being passed a string rather than an array

```
index.js:128 Warning: Failed prop type: Invalid prop `value` of type `string` supplied to `Multiselect`, expected `array`.
    in Multiselect (created by Uncontrolled(Multiselect))
    in Uncontrolled(Multiselect) (at slo-form.js:350)
    in div (at slo-form.js:348)
    in div (at slo-form.js:347)
    in SloForm (at slo-r-entity/index.js:382)
    in div (created by t)
    in div (created by t)
    in t (created by t)
    in t (created by Modal)
    in Modal (at slo-r-entity/index.js:375)
    in div (at slo-r-entity/index.js:340)
    in SLOREntityNedlet (at nr1-csg-slo-r-nerdlet/index.js:25)
    in Nr1CsgSloR
    in t
```

**Comment by doesterr** [comment]
I get the same exception:
`single-document.js:25 Uncaught (in promise) TypeError: _alerts.map is not a function`

Preventing me from setting up any SLO.

**Comment by ricegi** [comment]
Should be addressed with rel 1.4.6
